lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1230,8 +1230,8 @@ def _on_logging_relation_departed(self, event: RelationDepartedEvent):
             event: a `CharmEvent` in response to which the Loki
                 charm must update its relation data.
         """
-        if event.relation.data.get(event.relation.app):
-            self._remove_alert_rules_files(self.container)
+        if isinstance(event, RelationEvent):
```

Comment:
I'm a bit confused about this. How could it be anything other than a RelationEvent?

Reply:
You are right, a missing line from a big refactor that did not survive